packages/docs-page/server.js

```diff
@@ -3,18 +3,22 @@ import path from 'path'
 import validateFilePaths from '@hashicorp/react-docs-sidenav/utils/validate-file-paths'
 import validateRouteStructure from '@hashicorp/react-docs-sidenav/utils/validate-route-structure'
 import validateUnlinkedContent from '@hashicorp/react-docs-sidenav/utils/validate-unlinked-content'
-import {
-  loadVersionList,
-  loadVersionedDocument,
-  loadVersionedNavData,
-  getVersionFromPath,
-} from '@hashicorp/versioned-docs/server'
 import moize from 'moize'
-import { normalizeVersion } from '@hashicorp/versioned-docs/client'
+
 import renderPageMdx from './render-page-mdx'
+import { stripVersionFromPathParams, normalizeVersion } from './util'
+import { fetchNavData, fetchDocument, fetchVersionMetadataList } from './rpc'
```

Comment:
nit: RPC doesn't seem like the correct term to use here...what do you think about `content-api` or just `api` for this folder?

Reply:
`content-api` it is!